### PR TITLE
AGFEO.AGFEODashboard version 1.8.21

### PR DESCRIPTION
**Pull request by oneandonlyjason**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/27280)

### CODE REVIEW

File: manifests/a/AGFEO/AGFEODashboard/1.8.21/AGFEO.AGFEODashboard.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: AGFEO.AGFEODashboard
+PackageVersion: 1.8.21
+PackageLocale: en-US
+Publisher: AGFEO
+PackageName: AGFEO Dashboard
+License: proprietary
+ShortDescription: Dashboard zur Nutzung mit AGFEO Telefonanlagen
```
Comment (ItzLevvie) [inline manifests/a/AGFEO/AGFEODashboard/1.8.21/AGFEO.AGFEODashboard.locale.en-US.yaml:10]:
```
ShortDescription: Dashboard zur Nutzung mit AGFEO Telefonanlagen
```
Please write an english description.

File: manifests/a/AGFEO/AGFEODashboard/1.8.21/AGFEO.AGFEODashboard.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: AGFEO.AGFEODashboard
+PackageVersion: 1.8.21
+PackageLocale: en-US
+Publisher: AGFEO
+PackageName: AGFEO Dashboard
+License: proprietary
```
Comment (ItzLevvie) [inline manifests/a/AGFEO/AGFEODashboard/1.8.21/AGFEO.AGFEODashboard.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Please also sign the CLA: https://github.com/microsoft/winget-pkgs/pull/27280#issuecomment-915930312

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/27280#discussion_r705200056

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Missing: https://github.com/microsoft/winget-pkgs/pull/27280#discussion_r705200056

**Review by ItzLevvie — APPROVED** [review]

**Comment by oneandonlyjason** [comment]
> Missing: [#27280 (comment)](https://github.com/microsoft/winget-pkgs/pull/27280#discussion_r705200056)

I was not home earlier to change it sorry

**Pull request merged**